src/pages/calculator.tsx

```diff
@@ -0,0 +1,328 @@
+import FinalAmount from '@components/calculator/FinalAmount';
+import CTA from '@components/CTA';
+import {
+	FormInput,
+	FormInputWrapper,
+	FormRadioGroup,
+	FormRadioLabel,
+	FormRadioWrapper,
+	FormSelect,
+} from '@components/styled/Forms';
+import Link from 'next/link';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+type OnChangeType =
+	| React.ChangeEvent<HTMLInputElement>
+	| React.ChangeEvent<HTMLSelectElement>;
+
+type FormDataType = {
+	shape: string;
+	diameter: string;
```

Comment:
it should all be number, width, and length too

Reply:
Ok. Done